frag.d.ts

```diff
@@ -1,11 +1,15 @@
-import { ComponentOptions } from 'vue';
+import Vue, { ComponentOptions } from 'vue';
 
-type FragmentComponent = ComponentOptions<never, never, never, never, never, never>;
+type FragmentComponent<V extends Vue = Vue> = ComponentOptions<V, never, never, never, never, never>;
```

Comment:
What's the point/benefit of the `V` generic?

Reply:
No benefit really.. even `any` should work. But not `never`

Reply:
I think I originally had it as `never` because [none of these component methods](https://github.com/vuejs/vue/blob/6aa11872c88481dfa2da151536317176c48f226c/types/options.d.ts#L68-L123) are external so it wouldn't really matter.

Does having it as `never` break something?

Reply:
This one https://github.com/privatenumber/vue-frag/issues/63

Reply:
Nuxt extends the Component instance https://github.com/nuxt/nuxt.js/blob/777a4b7f5033c86c37cbd93008f3ca792e4af8bc/packages/types/app/vue.d.ts

That's why I added the `V` generic to just infer it

Reply:
Gotcha, thanks for the explanation

Reply:
Thought about this a little more.

Have you confirmed that it's absolutely necessary for the `V` generic for it to use Nuxt's Vue types?

I'm thinking that since they do declaration merging, when vue-frag imports the type from Vue, it will already have Nuxt's type extensions.